modules/genomic_browser/help/genomic_browser.md

```diff
@@ -0,0 +1,20 @@
+# Genomic Browser
+
+This module is intended to allow users to visualize the results of already analyzed genetic tests cross-linked with behavioral and imaging candidate data.
```

Comment:
Is this supposed to be the README.md? The help text is generally written in the second person addressed to users.